File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/UtpekingAvvisDto.java
package no.nav.melosys.eessi.controller.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class UtpekingAvvisDto {

    private String nyttLovvalgsland;
    private String begrunnelseUtenlandskMyndighet;
    private boolean vilSendeAnmodningOmMerInformasjon;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/UtpekingAvvisDto.kt
package no.nav.melosys.eessi.controller.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class UtpekingAvvisDto(
    var nyttLovvalgsland: String? = null,
    var begrunnelseUtenlandskMyndighet: String? = null,
    var vilSendeAnmodningOmMerInformasjon: Boolean = false
)
